manifest.yml

```diff
@@ -4,7 +4,7 @@ description: "This is the standard distribution of FreeRTOS."
 
 dependencies:
   - name: "FreeRTOS-Kernel"
-    version: "d3c289f"
```

Comment:
The top commit in the submodule updated is c3dc20f, there seems to be mismatch, is that correct?